aspnetcore/tutorials/razor-pages/page.md

```diff
@@ -1,34 +1,52 @@
 ---
-title: Part 3, scaffolded Razor Pages in ASP.NET Core
+title: Part 3, scaffolded Razor Pages
 author: rick-anderson
 description: Part 3 of tutorial series on Razor Pages.
 ms.author: riande
-ms.date: 08/17/2019
-no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+ms.date: 09/25/2020
+no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/razor-pages/page
 ---
 
 # Part 3, scaffolded Razor Pages in ASP.NET Core
 
-::: moniker range=">= aspnetcore-3.0"
-
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 This tutorial examines the Razor Pages created by scaffolding in the [previous tutorial](xref:tutorials/razor-pages/model).
 
-[!INCLUDE[View or download sample code](~/includes/rp/download.md)]
+::: moniker range=">= aspnetcore-5.0"
+
+[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie50) ([how to download](xref:index#how-to-download-a-sample)).
+
+::: moniker-end
+
+::: moniker range="< aspnetcore-5.0 >= aspnetcore-3.0"
+
+[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie30) ([how to download](xref:index#how-to-download-a-sample)).
+
+::: moniker-end
+
+::: moniker range=">= aspnetcore-3.0"
 
 ## The Create, Delete, Details, and Edit pages
 
 Examine the *Pages/Movies/Index.cshtml.cs* Page Model:
 
 [!code-csharp[](razor-pages-start/snapshot_sample3/RazorPagesMovie30/Pages/Movies/Index.cshtml.cs)]
 
-Razor Pages are derived from `PageModel`. By convention, the `PageModel`-derived class is called `<PageName>Model`. The constructor uses [dependency injection](xref:fundamentals/dependency-injection) to add the `RazorPagesMovieContext` to the page. All the scaffolded pages follow this pattern. See [Asynchronous code](xref:data/ef-rp/intro#asynchronous-code) for more information on asynchronous programming with Entity Framework.
+The constructor uses [dependency injection](xref:fundamentals/dependency-injection) to add the `RazorPagesMovieContext` to the page:
+
+[!code-csharp[](razor-pages-start/snapshot_sample3/RazorPagesMovie30/Pages/Movies/Index.cshtml.cs?name=snippet1?highlight=5)]
+
+See [Asynchronous code](xref:data/ef-rp/intro#asynchronous-code) for more information on asynchronous programming with Entity Framework.
+
```

Comment:
What happened to 

Razor Pages are derived from PageModel. By convention, the PageModel-derived class is called <PageName>Model.

Reply:
VS tab
![image](https://user-images.githubusercontent.com/3605364/98311879-8e5c1980-1f74-11eb-8757-379fdb146ea3.png)

The other combined tab
![image](https://user-images.githubusercontent.com/3605364/98311938-b186c900-1f74-11eb-922b-37c091ab837d.png)

Sure looks like duplication too me

Reply:
Work on this later, I think the review page is not rendering correctly. The 3.1 version shows it's not easy to move out.

Reply:
Now now, but I'd move 
## Add NuGet packages and EF tools to the top of the VS tab. It's probably better to add NuGet first anyway, that way you don't get compiler errors when you add the code.

Reply:
That seems like a quick/easy fix

Reply:
"Sure looks like duplication too me"   Yes, it looked to me like dupes in your doc to me as well. I didn't see any options around it at the time, so I figured you had a good reason for it. I had looked originally at moving  up the NuGet pages & EF tool instructions to the top before introducing the model class but the felt odd not to introduce the model class first.  However, I agree there is a benefit of making sure the customer has all that all in there before they attempt to compile.   Seems like you found a better solution, thanks.  I will hold off however as you ask.